cmd/algod/main.go

```diff
@@ -158,6 +158,17 @@ func run() int {
 	}
 	defer fileLock.Unlock()
 
+	// Delete legaxy indexer.sqlite file if it happens to exist
+	indexerDBPath := filepath.Join(absolutePath, genesis.ID(), "indexer.sqlite")
+
+	if _, idxFilePathErr := os.Stat(indexerDBPath); idxFilePathErr == nil {
```

Comment:
There's a helper function for this you could import
```suggestion
	if util.FileExists(indexerDBPath) {
```